test/integration/datadog_kafka-consumer/bats/kafka_consumer_config.bats

```diff
@@ -1,11 +1,11 @@
 @test "kafka.yaml exists" {
-  [ -f /etc/dd-agent/conf.d/kafka_consumer.yaml ]
+  [ -f /etc/datadog-agent/conf.d/kafka_consumer.yaml ]
```

Comment:
ditto, wouldn't the expected path here be: `/etc/datadog-agent/conf.d/kafka_consumer.d/conf.yaml`?

We should probably also put this in a constant variable so that we don't need to repeat it in multiple places on the file.

sidenote: kind of surprised we're using bats here!

Reply:
I've used constant wherever it was possible, but I think this bats (discovered `bats` this with this projects...) is out of scope :D